content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -172,4 +173,8 @@ specifically relevant in multi-tenant clusters if semi-trusted or untrusted user
 are allowed limited access to a system.
 
 One option for mitigation of this issue would be to use [resource quotas](/docs/concepts/policy/resource-quotas/#object-count-quota)
-to limit the quantity of objects which can be created.
\ No newline at end of file
+to limit the quantity of objects which can be created.
+
+## {{% heading "whatsnext" %}}
+* To learn more about RBAC, see [RBAC documentation](/docs/reference/access-authn-authz/rbac/).
+* To learn more about Authorizing Policies via RBAC, see [Authorizing Policies](/docs/concepts/security/pod-security-policy/#via-rbac).
```

Comment:
Hi @harshitasao 
I think we don't mention` Authorizing Policies via RBAC` because PodSecurityPolicy is deprecated and It will be removed in v1.25(current cycle).

Reply:
Oop's. I will fix it. Thank you.